Source/UIKit/UIBarItem.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UIBarItem.swift
+//  Rex
+//
+//  Created by Bjarke Hesthaven Søndergaard on 24/07/15.
+//  Copyright (c) 2015 Neil Pankey. All rights reserved.
+//
+
+import ReactiveCocoa
+import UIKit
+
+extension UIBarItem {
+    /// Wraps a uibaritem's `enabled` state in a bindable property.
```

Comment:
Yeah, I was wondering if it should be called rex_pressed to be similar to
the UIButton interface or keep it different because the underlying
functionality is not the same.
On fre. 24. jul. 2015 at 19.29 Neil Pankey <notifications@github.com> wrote:

> In Source/UIKit/UIBarItem.swift
> <https://github.com/neilpa/Rex/pull/22#discussion_r35445369>:
>
> > @@ -0,0 +1,19 @@
> > +//
> > +//  UIBarItem.swift
> > +//  Rex
> > +//
> > +//  Created by Bjarke Hesthaven Søndergaard on 24/07/15.
> > +//  Copyright (c) 2015 Neil Pankey. All rights reserved.
> > +//
> > +
> > +import ReactiveCocoa
> > +import UIKit
> > +
> > +extension UIBarItem {
> > +    /// Wraps a uibaritem's `enabled` state in a bindable property.
>
> Can you match the capitalization of the class name here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/neilpa/Rex/pull/22/files#r35445369>.
>